### PR TITLE
Add basic logging

### PR DESCRIPTION
**Pull request by chetankothari**
This is based on top of the API docs branch. Will be updated once that branch gets merged.

### CODE REVIEW

File: src/Account.js [hunk 1]
```diff
@@ -14,14 +21,28 @@ export default class Account {
     this._mpeContract = mpeContract;
   }
 
+  /**
+   * Returns the token balance available.
+   * @returns {Promise.<BigNumber>}
+   */
   async balance() {
+    logger.info('Fetching account balance', { tags: ['Account'] });
```
Comment (raamb) [inline src/Account.js]: Should be debug ?

File: src/Account.js [hunk 2]
```diff
@@ -84,6 +141,7 @@ export default class Account {
 
     return new Promise((resolve, reject) => {
       if(!receipt.status) {
+        logger.error('Transaction failed');
```
Comment (raamb) [inline src/Account.js]: Lets add more context such as printing the receipt object itself. Will make it easier to troubleshoot

File: src/Account.js (hunk 1 above)
Comment (chetankothari) [reply src/Account.js]: Yeah, I think we can mark a bunch of other `logs` as debug. I will follow the following rule to differentiate between `debug` and `info` logs

`info` -> Any critical operation which is likely to have side effects
`debug` -> All other logs

Does the above splitting look good?

Comment (raamb) [reply src/Account.js]: Sounds good

File: src/Account.js (hunk 2 above)
Comment (chetankothari) [reply src/Account.js]: @raamb addressed this in [commit](https://github.com/singnet/snet-sdk-js/pull/3/commits/338443f13682ebe1872121e0235c40839b5fd26a) and [commit](https://github.com/singnet/snet-sdk-js/pull/3/commits/c45e1d2fe2c94fedb4e1c867cdc3fd6b8e7807a9)

**Pull request merged**